index.php

```diff
@@ -33,6 +33,7 @@
         <div>
             <button class="button login-btn" onclick="loginNotice()">Login</button>
             <button class="button register-btn" onclick="location.href=('register.php')">register</button>
+            <button class="button logout-btn" onclick="logout()">Logout</button>
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider conditional rendering of logout button</b></div>
<div id="fix">

Consider adding a check for `isLoggedIn` before showing the `logout` button. Currently, the logout button is visible regardless of login status which may confuse users.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
            <?php if(isset($_SESSION['username'])): ?><button class="button logout-btn" onclick="logout()">Logout</button><?php endif; ?>
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Lux-Jason/rms_new/pull/6#issuecomment-2543456910>#da6ca6</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged